### DISCUSSION TITLE
How to get name of related collection

### DISCUSSION THREAD

**Discussion by jsx101** [directus/directus#16397] [Archive]
How do I get the name of a related collection when making a custom interface? The following is my code.

```
import { useStores } from '@directus/extensions-sdk';
const { useRelationsStore } = useStores();
const { collection, field } = toRefs(props);
const relationInfo = relationsStore.getRelationForField(collection, field);
const relatedCollection = relationInfo.relatedCollection;
```

I get this error in the Inspect Element console
![image](https://user-images.githubusercontent.com/40978303/201084009-cc42a00c-eb30-4e5a-aaa9-01b58609c40c.png)

Below are what I assume `collection` and `field` from `props` are and I've verified their values are correct

- collection - the name of the collection my custom interface is used in
- field - the name of the field that uses my custom interface

Am I doing this right?

**Comment by br41nslug** [comment]
The `getRelationForField` function expects its 2 parameters to  be strings not refs 😄 
Removing `const { collection, field } = toRefs(props);` and using `relationsStore.getRelationForField(props.collection, props.field);` should do the trick.

**Reply by jsx101** [reply]
Thank you so much. I got it working now!
